openedx_external_enrollments/tests/external_enrollments/tests_salesforce_external_enrollment.py

```diff
@@ -237,7 +239,11 @@ def test_get_courses_data(self, get_course_mock, get_course_key_mock, get_date_m
         course_mock = Mock()
         course_mock.end = now
         course_mock.display_name = 'test-course'
-        course_mock.other_course_settings = {'salesforce_data': {}}
+        course_mock.other_course_settings = {'salesforce_data': {
+            "Institution_Hidden": "HI",
+            "Program_of_Interest": "POI",
+        }}
```

Comment:
@Jacatove, please try to use only single quotes throughout the file.
```suggestion
        course_mock.other_course_settings = {
             'salesforce_data': {
                 'Institution_Hidden': 'HI',
                 'Program_of_Interest': 'POI',
             },
         }
```